src/xwayland-unmanaged.c

```diff
@@ -139,8 +140,8 @@ struct xwayland_unmanaged *
 xwayland_unmanaged_create(struct server *server,
 			  struct wlr_xwayland_surface *xsurface)
 {
-	struct xwayland_unmanaged *unmanaged;
-	unmanaged = calloc(1, sizeof(struct xwayland_unmanaged));
+	struct xwayland_unmanaged *unmanaged
+		= xzalloc(sizeof(struct xwayland_unmanaged));
```

Comment:
nit: We are usually having the `=` at the end of the first line

Reply:
That reminds me.... haven't run `checkpatch.pl` for a while :smile: 
Incoming